src/components/date-picker/date-picker--range.html

```diff
@@ -0,0 +1,24 @@
+<!-- Ranged -->
+<div class="bx--form-item">
+  <div data-date-picker data-date-picker-type="range" class="bx--date-picker bx--date-picker--range">
+    <div class="bx--date-picker__container">
```

Comment:
FWIW, I thought we were doing it the way @tw15egan said. @chrisdhanaraj @iangfleming